internal/compiler/parser/expressions.rs

```diff
@@ -438,6 +441,37 @@ fn parse_tr(p: &mut impl Parser) {
     p.expect(SyntaxKind::RParent);
 }
 
+fn parse_markdown(p: &mut impl Parser) {
```

Comment:
Please add some basic tests to this function. You can see with `parse_tr()` right above that there's a framework for that. That way we can catch any future refactoring mistakes earlier.